src/raftstore/store/peer.rs

```diff
@@ -796,6 +734,7 @@ impl Peer {
 
         // If applied index's term is differ from current raft's term, leader transfer
         // must happened, if read locally, we may read old value.
+        // TODO: use raft_log's applied_index to check instead.
```

Comment:
`applied_index_term` of `PeerStorage` may be not updated in time, so some requests that can be performed locally has to fallback to quorum read when new election happens.